docs/codeql/codeql-language-guides/codeql-library-for-javascript.rst

```diff
@@ -179,8 +179,6 @@ As an example of a query using only lexical information, consider the following
    from HtmlLineComment c
    select c, "Do not use HTML comments."
 
-➤ `See this in the query console on LGTM.com <https://lgtm.com/query/686330023/>`__. When we ran this query on the *mozilla/pdf.js* project in LGTM.com, we found three HTML comments.
-
 Syntactic level
 ~~~~~~~~~~~~~~~
 
```

Comment:
I have a query about the note in line 231 about minified code.

> By default, LGTM filters out alerts in minified top-levels, since they are often hard to interpret. When writing your own queries in the LGTM query console, this filtering is *not* done automatically, so you may want to explicitly add a condition of the form `and not e.getTopLevel().isMinified()` or similar to your query to exclude results in minified code.

Does code scanning also automatically filter out alerts in minified top-levels? Should people remember to use and `not e.getTopLevel().isMinified()` or similar when they write a query in VS Code?

Reply:
Code-scanning does filter out such alerts (I just tested), although I’m not sure how it works.

Reply:
Many thanks for checking @erik-krogh. Presumably we don't do this filtering in VS Code, in the same way as we don't in the LGTM console?

Reply:
Correct, there is no filtering in VS Code.

Reply:
Thank you ✨

Reply:
I've pushed a commit that updates the note accordingly.